pkg/services/dashboards/dashboard.go

```diff
@@ -24,6 +24,7 @@ type DashboardService interface {
 	SaveDashboard(ctx context.Context, dto *SaveDashboardDTO, allowUiUpdate bool) (*Dashboard, error)
 	SearchDashboards(ctx context.Context, query *FindPersistedDashboardsQuery) (model.HitList, error)
 	CountInFolder(ctx context.Context, orgID int64, folderUID string, user identity.Requester) (int64, error)
+	GetDashboardsSharedWithUser(ctx context.Context, user identity.Requester) ([]*Dashboard, error)
```

Comment:
Why a specific service operation??

Reply:
It's not necessary now, but I put it here in case we need a separate API endpoint for fetching "shared" dashboards.